lib/msal-browser/src/app/ClientApplication.ts

```diff
@@ -1148,15 +1148,17 @@ export abstract class ClientApplication {
             browserState
         );
 
-        const authenticationScheme = request.authenticationScheme || AuthenticationScheme.BEARER;
+        // Set authenticationScheme to BEARER if not explicitly set in the request
+        if (!request.authenticationScheme) {
+            request.authenticationScheme = AuthenticationScheme.BEARER;
+        }
```

Comment:
Can you please add verbose log messages to both sides of this conditional?

Reply:
Done